duckduckgo/macos-browser
# Secure Vault additions

Task/Issue URL: https://app.asana.com/0/72649045549333/1199992415799651/f
Tech Design URL:
CC:

**Description**:

⚠️ Note: This PR depends on https://github.com/duckduckgo/BrowserServicesKit/pull/28 being merged.

This PR adds support for new Autofill data types: credit cards, identities, and notes. Autofill is only supported for credit cards right now, so Identities are disabled in release builds until we get that working.

**Steps to test this PR**:
1. Try creating each of the new data types 
1. Try creating a login to ensure that flow still works
1. Try editing each of the new data types
1. Try editing a login
1. Create a credit card, and visit a site with a credit card form to test autofill. I've found Amazon to work well enough for this, and it's what Emanuele's been testing with

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
